dm/worker/worker.go

```diff
@@ -403,6 +405,11 @@ func (w *Worker) StartSubTask(cfg *config.SubTaskConfig, expectStage pb.Stage, n
 
 	// directly put cfg into subTaskHolder
 	// the unique of subtask should be assured by etcd
+	if w.logConfig != nil && cfg.LogFile == "" {
+		cfg.LogLevel = w.logConfig.Level
```

Comment:
can lightning use external logger? so we won't bother adding a logConfig

dumpling's example https://github.com/pingcap/dm/blob/6ad2f890822495986e7d98a55d5202b72a8442b9/dumpling/dumpling.go#L234